Source/ASDisplayNode.mm

```diff
@@ -828,6 +830,94 @@ - (void)nodeViewDidAddGestureRecognizer
   _flags.viewEverHadAGestureRecognizerAttached = YES;
 }
 
+#pragma mark UIResponder
```

Comment:
IMO this should go in another file, since ASDisplayNode.mm is quite large - it seems very appropriate for UIViewBridge or perhaps we could introduce another one.

Not a blocking change (feel free to land this if you're just eager to get it in!)

Reply:
We are currently have the event handling `touchesBegan:` etc. in `ASDisplayNode.mm`, eventually we should move both of them out too in a follow up diff.